agents/e2e-verifier.md

```
@@ -0,0 +1,58 @@
+- `test.skip` / コメントアウト / 到達不能なコードでテストが実質無効化されていないか
+
+### Step 3: テストの有効性（実行確認・任意のミューテーション）
+
+- 可能なら対象E2Eを実行し、実際にパスする（=実行される）ことを確認する
+- 重要フローについては**ミューテーション**（故意の不具合注入）でテストが確かに失敗することを確認する
+  - 対象コードに一時的な不具合を注入 → テスト実行 → 失敗することを確認
+  - **注入は必ず元に戻す**（`git checkout -- <file>` 等で確実に復元し、復元を確認する）
+
+## 報告
+
+```
```

markdownlint-cli2 (0.22.1)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)